tasks/files/snapshot.py

```diff
@@ -778,28 +1310,39 @@ def print_result(rc, message):
         "-r",
         "--requiredspace",
         dest="required_space",
+        nargs="?",
         required=False,
-        type=int,  # choices=range(10,100)
```

Comment:
why not use `type=int`?  The default value is an `int` - will this argument hold values other than `int`?

Reply:
yeah, type=int is better.  I was trying to allow for -r and no value associated.  but I updated how the cmd is constructed instead.  I'll push changes when I'm done reviewing other problems.

Reply:
I removed this change.